mt7697/README.md

```diff
@@ -0,0 +1,18 @@
+# LINE Things Starter for Linkit 7697
+
+## Requirements
+* [Arduino IDE](https://www.arduino.cc/en/Main/Software)
+* [Linkit 7697](https://labs.mediatek.com/zh-tw/chipset/MT7697)
```

Comment:
Could you change this link to English page?

Reply:
Done with new commit.